test/Microsoft.AspNetCore.OData.E2E.Tests/AutoExpand/AutoExpandTests.cs

```diff
@@ -488,5 +538,23 @@ public async Task PutCustomer_AutoExpandNavigationProperties()
                 }
             }
         }
+
+        private async Task VerifyNavigationPropertiesAndLevels(HttpResponseMessage response, int propCount)
```

Comment:
```suggestion
        private async Task VerifyNavigationProperties(HttpResponseMessage response, int propCount)
```

I also wish the method would accept a `JObject` instead of `HttpResponseMessage`, just like the other "Verify..." methods

Reply:
Applied